src/Encoder/MaskUtil.php

```diff
@@ -204,7 +204,7 @@ public static function getDataMaskBit(int $maskPattern, int $x, int $y) : bool
                 break;
 
             case 4:
-                $intermediate = (BitUtils::unsignedRightShift($y, 1) + ($x / 3)) & 0x1;
+                $intermediate = (BitUtils::unsignedRightShift($y, 1) + (int)($x / 3)) & 0x1;
```

Comment:
Please add a space after the cast operator, otherwise this is fine.

Reply:
Done.